root/static/scripts/common/components/CommonsImage.js

```diff
@@ -52,4 +52,6 @@ class CommonsImage extends React.Component<Props, State> {
   }
 }
 
-export default hydrate<Props>('div.commons-image', CommonsImage, minimalEntity);
+export default hydrate<Props>(
+  'div.commons-image', CommonsImage, minimalEntity,
```

Comment:
When we have to move arguments to the next line (so no argument starts on the same line as the opening `(`), I'd actually prefer putting them all on separate lines. It'd seem easier to read to me that way, because if I see an initial line break, then I instinctively look for an argument per line and don't expect them to share a line.

If we enabled https://eslint.org/docs/rules/function-call-argument-newline and set it to "consistent," I think the style used here would even be an error, since the first argument has a line break before it while the others don't. (But this common style would still be allowed:

```JS
foo('bar', {
  baz: true,
});
```
since none of the arguments start with line breaks there.)

(This isn't the only instance of this change, I brought it up because there are many.)